test/worker-runner.js

```diff
@@ -22,7 +22,7 @@ self.onmessage = function (e) {
     "level1/html": require("../test/level1/html"), // ok
     "level2/core": require("../test/level2/core"), // ok
     "level2/html": require("../test/level2/html"), // 6/708
```

Comment:
Curious if level2/html got any better? Wonder what makes it fail so hard when level1/html works.